app/styles/global.scss

```diff
@@ -70,7 +97,25 @@ $vz-sand: #eee6de;
   max-height: 25rem;
 }
 
-.card-header,
+.card {
+  border: 1px solid #e5e7eb !important;
+  box-shadow: 1px 1px 0 rgba(0, 0, 0, 0.1), 0 1px 2px -1px rgba(0, 0, 0, 0.1);
+}
+
+.card-header {
+  padding: 1rem;
+  border: none !important;
+  background-color: inherit !important;
+}
+
+h5.card-title {
+  font-size: 1.3rem !important;
+  font-weight: bold;
+}
+
 .card-footer {
-  border: none;
+  background-color: inherit !important;
+  border: none !important;
```

Comment:
minimalized card styles, with the header/footer borders and background removed
<img width="491" alt="Screenshot 2025-01-28 at 3 26 38 PM" src="https://github.com/user-attachments/assets/4fd81384-0aec-4d5e-a40d-88cc85c2ee29" />